lib/Controller/WizardController.php

```diff
@@ -136,6 +133,21 @@ public function staticSlide($name, $params) {
 			return null;
 		}
 
+		$this->initialState->provideInitialState(
+			'desktop',
+			$this->config->getSystemValue('customclient_desktop', $this->theming->getSyncClientUrl())
+		);
+
+		$this->initialState->provideInitialState(
+			'android',
+			$this->config->getSystemValue('customclient_android', $this->theming->getSyncClientUrl())
+		);
+
+		$this->initialState->provideInitialState(
+			'ios',
+			$this->config->getSystemValue('customclient_ios', $this->theming->getSyncClientUrl())
+		);
```

Comment:
it looks like we need to have a template response and not a jsonresponse in order to make it work :/